src/app/shared/components/multi-select/multi-select.component.scss

```diff
@@ -17,10 +17,6 @@ label {
   border: 1px solid var(--gray-color);
 }
 
-.multi-select:hover .multi-select-dropdown {
```

Comment:
What is the reasoning for removing this?

Reply:
To remove the logic witch show the dropdown on hover and add this type of logic to be only in click.

Reply:
Ahh, yeah, I missed that :D